### ISSUE TITLE
Can't participate in battles and dragons jumping levels

### ISSUE THREAD

**Issue by Starlight5351** [SoDOff-Project/sodoff#15]
I don't really know if this is a proper place to ask this so if not I do sincerely apologize. I'm hoping someone can help me with my issues as I've been having them since downloading SoDOff back in July. I can't participate in battles because after the ship is defeated and the "Victory" sign pops up another ship appears and the "Victory" sign won't go away. My dragons also jump levels when I get off the game and get back on, particularly if I feed them. For example, my dragon might be level 15 when I get off the game and then when I get on again at a later time my dragon has jumped to level 17. I've done everything I can to fix this myself: uninstalling and reinstalling; uninstalling, clearing cache, and reinstalling; creating a new viking on my account; creating a new account; deleting my temp and junk folders and files. Absolutely nothing works. I had a post up on the SchoolofDragons subreddit about it but it got taken down by Reddit's filters and I can't make a new post because I tried and Reddit's filters immediately took it down. I don't know what else to do. I'm desperate for help so I decided to try here. I hope someone can help me. One more thing, I'm using a Windows HP 8.1 laptop to play the game. The fact that Windows 8.1 is now outdated doesn't have anything to do with it does it? I mean, the original game worked fine but I guess you never know.

**Comment by rpaciorek** [comment]
https://github.com/Aidanamite/Bugfixes ← this should fix Victory screen issue, dragon leveling too fast is a know issue (related to achievements system)

**Comment by Starlight5351** [comment]
Thanks! This may sound stupid but I'm not tech savvy at all so is there anything new I need to download so the battle issue will be fixed or should it just automatically be fixed when I get on the game?

**Comment by Starlight5351** [comment]
I finally joined the discord and after a little bit of reading figured out what I needed to download in order to get bug fixes. Tested it out and it's fixed. Thanks for the fix! Also just in case you or someone else sees this I was wanting to know if the dragon leveling too fast issue was ever going to be fixed. Not trying to be pushy or rush you guys of course. I was just curious if it would ever be fixed at some point.
